HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
+Ethereum addresses, and permits the use of this `alias` for all operations supported by `alias` today. This new
+capability would allow an account with, for example, a threshold key to have an `ECDSA` derived alias such that
+a MetaMask user could transfer funds to that account.
+
+This HIP expands support in the auto-create flow for `ECDSA` derived addresses to be used as the `alias`. It also
+adds support to `CryptoCreate` transactions for creation with either a key-based `alias` or, an `ECDSA` derived
+`alias`.
 
 ## Rationale
 
-With [HIP 32](https://hips.hedera.com/hip/hip-32) it’s possible to send HBARs in a `CryptoTransfer` to an account before it exists on the ledger by providing an alias using a cryptographic public key (`ED25519` or `ECDSA`) of the `public-key` format. 
-Using this, the ledger will confirm the uniqueness of the alias and create an account that can be referenced via this accounts public key alias.
-
-However, the ability to create accounts or refer to accounts with an alias is limited and confusing for Smart Contract user experiences since
-
-- Aliases can only be set using a `CryptoTransfer` transaction, whereas wallets would prefer to do this in a `CryptoCreate` transaction in addition to a `CrytoTransfer` transaction.
-- Users who only know their `public-address` must find innovative ways to find their public key or safely export their private key to extract the public key for use in the `CryptoTransfer`.
-- HBAR or token transfers using an alias only support the `public-key` alias format and not the `public-address` format.
-
-Notably, most ledgers in the market utilize the `ECDSA` key for accounts, especially where an `EVM` is concerned. In fact to completely support all Smart Contract scenarios on the `EVM` used by Hedera, an account must possess an alias based on its `ECDSA` public key that dictates the address value observed by teh EVM in transactuions. 
+Fundamental in the design of the Hedera network are the concepts of _accounts_, account _identifiers_, and _keys_. These
+are fundamentally different entities. An `Account` must have a key. The key for the account may be a simple key
+(typically, an `ED25519` public key), or it may be a compound key, such as a list of keys all of which must sign each
+transaction, or a threshold key such that N of the keys must be signed. These may be nested to create very rich and
+powerful accounts requiring multiple-signatures. Further, an `Account` may change the keys on the account over time.
+
+An `Account` also has a system-assigned account identifier of the form `shard.realm.num`, where each value is an 8-byte
+number. An `Account` may also be referred to with an identifier of the form `shard.realm.alias`, where the alias is of one of
+the following forms:
+
+1. `long-zero alias` - This is the Hedera account number styled as an Ethereum account hex string. The
```

Comment:
Suggest `entity num alias` or `entity id alias`

Reply:
I went with `account num alias` since I think it only applies to accounts (... and contracts... but they are accounts).